main.py

```diff
@@ -0,0 +1,128 @@
+import cv2
+from matplotlib import image
+import mediapipe as mp
+import numpy as np
+
+mp_drawing = mp.solutions.drawing_utils
+mp_drawing_styles = mp.solutions.drawing_styles
+mp_holistic = mp.solutions.holistic
+mp_pose = mp.solutions.pose
+cap = cv2.VideoCapture('D:\\Tenis_04_07\\Wideonly\\zverev.mp4')
```

Comment:
If you are comitting something to the branch don't forget to remove all your local paths. Actually if script requires a path or any other argument it should be done by ArgumentParser, eg.
```
# myls.py
# Import the argparse library
import argparse

import os
import sys

# Create the parser
my_parser = argparse.ArgumentParser(description='List the content of a folder')

# Add the arguments
my_parser.add_argument('Path',
                       metavar='path',
                       type=str,
                       help='the path to list')

# Execute the parse_args() method
args = my_parser.parse_args()

input_path = args.Path

if not os.path.isdir(input_path):
    print('The path specified does not exist')
    sys.exit()

print('\n'.join(os.listdir(input_path)))
```